lang/en/docs/workspaces.md

```diff
@@ -0,0 +1,92 @@
+---
+id: docs_workspaces
+guide: docs_workspaces
+layout: guide
+---
+
+{% include vars.html %}
+
+Workspace are a new way to setup your package architecture that's available by default starting from Yarn 1.0. It allows you to setup multiple packages in such a way that you only need to run `yarn install` once to install all of them in a single pass.
+
+### Why would you want to do this?
+
+- Your dependencies can be linked together, which means that a workspace can depend on another and always use the most up-to-date version of it, without having to run `yarn install` again like with the `file:` protocol, and with better safety mechanisms than with `yarn link`.
```

Comment:
Better to split into two sentences:

Your dependencies can be linked together, which means a workspace can depend on another and use the latest version of it from the monorepo. This is also a safer mechanism than `yarn link` since yarn now keeps track of actual dependencies along with versions even if they are linked locally.